octobot_trading/producers/prices_updater.py

```diff
@@ -37,17 +37,17 @@ def __init__(self, channel):
         self.refresh_time = MarkPriceUpdater.MARK_PRICE_REFRESH_TIME
 
     async def start(self):
-        if not self.channel.exchange_manager.is_future:
-            await self.subscribe()
-        elif self._should_run():
-            refresh_threshold = self.channel.exchange_manager.get_rest_pairs_refresh_threshold()
-            if refresh_threshold is RestExchangePairsRefreshMaxThresholds.MEDIUM:
-                self.refresh_time = 12
-            elif refresh_threshold is RestExchangePairsRefreshMaxThresholds.SLOW:
-                self.refresh_time = 17
-            if self.channel.is_paused:
-                await self.pause()
-            else:
+        refresh_threshold = self.channel.exchange_manager.get_rest_pairs_refresh_threshold()
+        if refresh_threshold is RestExchangePairsRefreshMaxThresholds.MEDIUM:
+            self.refresh_time = 12
+        elif refresh_threshold is RestExchangePairsRefreshMaxThresholds.SLOW:
+            self.refresh_time = 17
```

Comment:
Are refresh times similar for future comparing to spot markets ?

Reply:
I think it should be similar.